src/components/Score.js

```diff
@@ -3,16 +3,40 @@ import { browserHistory } from 'react-router';
 
 import ScoreInput from './ScoreInput.js';
 
+const pushPayload = {
+  sessionId: null,
+  scopes: {}
+};
+
 const Score = React.createClass({
+  updatePushPayload(scope, score, comment) {
+    pushPayload.sessionId = this.props.params.sessionId;
+    pushPayload.scopes[scope] = pushPayload.scopes[scope] || {};
+    pushPayload.scopes[scope].score = score;
+    pushPayload.scopes[scope].comment = comment;
+  },
   submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+    // TODO: validation - pk2
+    fetch('http://localhost:3000/api/score', {
+      method: 'post',
```

Comment:
If our score object had a `save` method, we could delegate to it here. 
At some point we also need to abstract the `localhost:3000` bit. But I think we can make a separate issue to track that.